_includes/v22.2/orchestration/kubernetes-limitations.md

```diff
@@ -26,4 +26,6 @@ When starting Kubernetes, select machines with at least **4 vCPUs** and **16 GiB
 
 #### Storage
 
-At this time, orchestrations of CockroachDB with Kubernetes use external persistent volumes that are often replicated by the provider. Because CockroachDB already replicates data automatically, this additional layer of replication is unnecessary and can negatively impact performance. High-performance use cases on a private Kubernetes cluster may want to consider using [local volumes](https://kubernetes.io/docs/concepts/storage/volumes/#local).
+- At this time, orchestrations of CockroachDB with Kubernetes use external persistent volumes that are often replicated by the provider. Because CockroachDB already replicates data automatically, this additional layer of replication is unnecessary and can negatively impact performance. High-performance use cases on a private Kubernetes cluster may want to consider using [local volumes](https://kubernetes.io/docs/concepts/storage/volumes/#local).
+
+- Service Name Indication (SNI) is an extension to the TLS protocol which allows a client to indicate which hostname it is attempting to connect to at the start of the TCP handshaking process. The server can present multiple certificates on the same IP address and TCP port number, and one server can serve multiple secure websites or API services even if they use different certificates. Due to its order of operations, the PostgreSQL wire protocol's implementation of TLS is not compatible with SNI in Kubernetes.
```

Comment:
We need to be careful with the nuances of SNI support here, especially now that we are relying on SNI (in kubernetes) for serverless (#15653). The thing that's incompatible is not "SNI in Kubernetes", it's SNI-based routing in the Kubernetes ingress controller. 

We can also give a positive recommendation: use a TCP load balancer that is not shared with other services instead of using SNI on a shared ingress controller.

Reply:
Thanks for the context and the positive recommendation. I'll integrate it.

Reply:
Done